examples/energy-management-app/energy-management-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -33,6 +34,31 @@ using chip::Protocols::InteractionModel::Status;
  */
 CHIP_ERROR GetEpochTS(uint32_t & chipEpoch);
 
+/**
+ * @brief   Helper function to get current timestamp and work out the day of week based on localtime
+ *
+ * @param   reference to hold the day of week as a bitmap
+ *
+ * Sunday = 0x01, Monday = 0x01 ... Saturday = 0x40 (1<<6)
+ */
+CHIP_ERROR GetDayOfWeekNow(uint8_t & dayOfWeekMap);
+
+/**
+ * @brief   Helper function to get current timestamp and work out the day of week
+ *
+ * NOTE that the time_t is converted using localtime to provide the timestamp
+ * in local time. If this is not supported on some platforms an alternative
+ * implementation may be required.
+ *
+ * @param   unixEpoch (as time_t)
+ *
+ * @return  bitmap value for day of week
+ * Sunday = 0x01, Monday = 0x01 ... Saturday = 0x40 (1<<6)
+ */
+uint8_t GetDayOfWeekUnixEpoch(time_t unixEpoch);
+
+CHIP_ERROR GetMinutesPastMidnight(uint16_t & minutesPastMidnight);
```

Comment:
please add documentation for this: minutes past midnight since when? Is this using the current time?

Reply:
Documentation added.